src/test/java/shapes/IsoscelesTriangleTest.java

```diff
@@ -0,0 +1,209 @@
+package shapes;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for IsoscelesTriangle.
+ */
+public class IsoscelesTriangleTest {
+    private final double DELTA = 1e0;
```

Comment:
W pozostałych testach korzystamy z 
```suggestion
    private final double DELTA = 1e-2;
```